kod/object/passive/spell/jala.kod

```diff
@@ -160,11 +160,22 @@ messages:
          if event = EVENT_DAMAGE
             OR event = EVENT_ATTACK
             OR event = EVENT_RUN
-            OR (event = EVENT_CAST AND Send(Send(SYS, @GetSettings), @CanRecastJalaWithNecklace))
+            OR (event = EVENT_CAST AND Send(Send(SYS, @GetSettings), @CanRecastJalaWithNecklace)
+               AND NOT IsClass(what,&Jala))
          {
             return FALSE;
          }
 
+         if event = EVENT_USE & usewhat <> $
```

Comment:
& is bitwise and.  You want AND instead.

Reply:
Thanks - fixed.